code_annotations/contrib/config/feature_toggle_annotations.yaml

```diff
@@ -9,6 +9,7 @@ annotations:
     - ".. documented_elsewhere:":
     - ".. documented_elsewhere_name:":
   feature_toggle:
+    # See annotation format documentation: https://edx-toggles.readthedocs.io/en/latest/how_to/documenting_new_feature_toggles.html
```

Comment:
I just realized we never discussed documented_elsewhere. Is it used? I feel like it was for issue of settings being in both LMS and CMS? Reminder to remove or document in how to at some point. Maybe add a note to the Confluence page? Should we track Open Questions there?

Reply:
AFAIK this is not used anywhere. Pinging @nedbat who is the original author: https://github.com/edx/edx-platform/pull/19715

> Maybe add a note to the Confluence page? Should we track Open Questions there?
Let's keep this comment open until we figure out a resolution.

Reply:
I remember it coming up in the Hackathon. I think it will come up again and we can discuss then. We should probably just remove this if it isn’t used.

Reply:
ok I'll open a new PR.